OpenOversight/app/models/database.py

```diff
@@ -620,6 +591,211 @@ class Image(BaseModel, TrackUpdates):
 )
 
 
+class Incident(BaseModel, TrackUpdates):
+    __tablename__ = "incidents"
+
+    id = db.Column(db.Integer, primary_key=True)
+    date = db.Column(db.Date, unique=False, index=True)
+    time = db.Column(db.Time, unique=False, index=True)
+    report_number = db.Column(db.String(50), index=True)
+    description = db.Column(db.Text(), nullable=True)
+    address_id = db.Column(
+        db.Integer, db.ForeignKey("locations.id", name="incidents_address_id_fkey")
+    )
+    address = db.relationship(
+        "Location",
+        backref=db.backref("incidents", cascade_backrefs=False),
+        lazy="joined",
+    )
```

Comment:
Addresses this periodic error: `sqlalchemy.orm.exc.DetachedInstanceError: Parent instance <Incident at 0xffff731ba510> is not bound to a Session; lazy load operation of attribute 'address' cannot proceed (Background on this error at: h...`